github_activity/cli.py

```diff
@@ -223,31 +225,37 @@ def main():
         ignored_contributors=args.ignore_contributor,
     )
 
-    if args.all:
-        md = generate_all_activity_md(args.target, **common_kwargs)
+    # Wrap in a try/except so we don't have an ugly stack trace if there's an error
+    try:
+        if args.all:
+            md = generate_all_activity_md(args.target, **common_kwargs)
```

Comment:
This is all just tab-adding. The only difference is the try/except statement, so that if valueerrors happen down below we can display them nicely.